<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MuseesMainWidget</class>
 <widget class="QWidget" name="MuseesMainWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb( 	207, 181, 59);</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="label_nomMusee">
         <property name="text">
          <string>Nom du mus√©e</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QGridLayout" name="gridLayout">
         <item row="2" column="0">
          <widget class="QLabel" name="gauche_fermeture">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="gauche_horaires">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLabel" name="droite_adresse">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLabel" name="droite_horaires">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLabel" name="droite_fermeture">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="gauche_adresse">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="gauche_nocturnes">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QLabel" name="droite_nocturnes">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="0" column="0">
          <widget class="QLabel" name="label_photo1">
           <property name="text">
            <string>Photo 1</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLabel" name="label_photo2">
           <property name="text">
            <string>Photo 2</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_photo3">
           <property name="text">
            <string>Photo 3</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLabel" name="label_photo4">
           <property name="text">
            <string>Photo 4</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLabel" name="label_map">
       <property name="text">
        <string>Map</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
